bindings/csharp/Regorus/Regorus.cs

```diff
@@ -0,0 +1,305 @@
+using System.Runtime.InteropServices;
```

Comment:
[nit] standard convention for C# is 4 spaces rather than tabs for whitespace

Reply:
Fixed. Needed to tweak a few vscode settings.